tests/integration/test_wazuh_db/test_sync_agent_groups_get.py

```diff
@@ -0,0 +1,136 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+           Created by Wazuh, Inc. <info@wazuh.com>.
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+type: integration
+brief: Wazuh-db is the daemon in charge of the databases with all the Wazuh persistent information, exposing a socket
+       to receive requests and provide information. The Wazuh core uses list-based databases to store information
+       related to agent keys, and FIM/Rootcheck event data.
+       This test checks the usage of the sync-agent-groups-get command used to allow the cluster getting the
+       information to be synchronized..
+tier: 0
+modules:
+    - wazuh_db
+components:
+    - manager
+daemons:
+    - wazuh-db
+os_platform:
+    - linux
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - CentOS 6
+    - Ubuntu Focal
+    - Ubuntu Bionic
+    - Ubuntu Xenial
+    - Ubuntu Trusty
+    - Debian Buster
+    - Debian Stretch
+    - Debian Jessie
+    - Debian Wheezy
+    - Red Hat 8
+    - Red Hat 7
+    - Red Hat 6
+references:
+    - https://documentation.wazuh.com/current/user-manual/reference/daemons/wazuh-db.html
+tags:
+    - wazuh_db
+'''
+import os
+import time
+import pytest
+import yaml
+import json
+from wazuh_testing.tools import WAZUH_PATH
+from wazuh_testing.wazuh_db import (query_wdb, insert_agent_into_group, clean_agents_from_db,
+                                    clean_groups_from_db, clean_belongs)
+from wazuh_testing.tools.file import get_list_of_content_yml
+
+# Marks
+pytestmark = [pytest.mark.linux, pytest.mark.tier(level=0), pytest.mark.server]
```

Comment:
Could you use marks defined on `deps\wazuh_testing\wazuh_testing\modules\ __init__.py`, please?

Reply:
Done in commit [aa349](https://github.com/wazuh/wazuh-qa/pull/2626/commits/aa3490433086795891eb1e22b27d58912006f9c5)